Composer/packages/client/src/components/BotRuntimeController/BotController.tsx

```diff
@@ -232,6 +232,7 @@ const BotController: React.FC<BotControllerProps> = ({ onHideController, isContr
                 },
               },
             }}
+            id={'startbot'}
```

Comment:
While we are at it would it be better to give it a more specific name? startBotPanelElement or something like that?

Reply:
Done!